vitest.setup.ts

```diff
@@ -34,6 +37,20 @@ vi.stubGlobal('useRuntimeConfig', () => ({
   },
 }))
 
+export const mockI18n = vi.fn((options?: NuxtI18nOptions) => ({
+  locale: options?.defaultLocale || 'en',
```

Comment:
Our fallback locale in `nuxt.config.ts` is German. We should either switch that to English, or the fallback here to German.